addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -112,6 +116,10 @@ class StudyService extends Service {
         throw this.boom.forbidden(`Not authorized to add study related to project "${projectId}"`);
       }
       await projectService.mustFind(requestContext, { id: rawData.projectId });
+      // Verify user is not trying to create resources for non-Open data studies
+      if (rawData.resources && rawData.resources.length > 0) {
```

Comment:
You can use
if (!_.isEmpty(rawData.resources)) {
}

Reply:
Done